bokeh/charts/__init__.py

```diff
@@ -1,21 +1,23 @@
 
-from .builder.area_builder import Area
-from .builder.donut_builder import Donut
-from .builder.dot_builder import Dot
-from .builder.line_builder import Line
-from .builder.step_builder import Step
-from .builder.histogram_builder import Histogram
-from .builder.bar_builder import Bar
-from .builder.scatter_builder import Scatter
-from .builder.boxplot_builder import BoxPlot
-from .builder.timeseries_builder import TimeSeries
-from .builder.heatmap_builder import HeatMap
-from .builder.horizon_builder import Horizon
+from .builder.area_builder import Area; Area
+from .builder.donut_builder import Donut; Donut
+from .builder.dot_builder import Dot; Dot
+from .builder.line_builder import Line; Line
+from .builder.step_builder import Step; Step
+from .builder.histogram_builder import Histogram; Histogram
+from .builder.bar_builder import Bar; Bar
+from .builder.scatter_builder import Scatter; Scatter
+from .builder.boxplot_builder import BoxPlot; BoxPlot
+from .builder.timeseries_builder import TimeSeries; TimeSeries
+from .builder.heatmap_builder import HeatMap; HeatMap
+from .builder.horizon_builder import Horizon; Horizon
 
-from ._chart import Chart
-from ._data_adapter import DataAdapter
+from ._chart import Chart; Chart
+from ._data_adapter import DataAdapter; DataAdapter
 
-from ..models import ColumnDataSource
+from ..models import ColumnDataSource; ColumnDataSource
 from ..io import (
     curdoc, cursession, output_file, output_notebook, output_server, push,
     reset_output, save, show, gridplot, vplot, hplot)
+(curdoc, cursession, output_file, output_notebook, output_server, push,
+ reset_output, save, show, gridplot, vplot, hplot)
```

Comment:
I can't think of anything.  A blank like then comment "# silence pyflakes" above the tuple might be nice but that's about it.